library/core/src/main/java/com/google/android/exoplayer2/upstream/RawResourceDataSource.java

```diff
@@ -58,7 +58,7 @@ public static Uri buildRawResourceUri(int rawResourceId) {
     return Uri.parse(RAW_RESOURCE_SCHEME + ":///" + rawResourceId);
   }
 
-  private static final String RAW_RESOURCE_SCHEME = "rawresource";
+  /* package */ static final String RAW_RESOURCE_SCHEME = "rawresource";
```

Comment:
This may as well be public (e.g. If someone wants to implement their own variant of a default data source outside of this package, it would be useful to be able to access this).

Reply:
Tried to keep it as close as original source, but yes public static will be happy for everyone ;D.